APIs/openEO/Collections.qmd

```diff
@@ -0,0 +1,25 @@
+---
+title: "Data collections"
+format:
+  html:
+    code-fold: true
+jupyter: python3
+---
+
+In the Earth Observation domain, different terms are used to describe EO data(sets). In openEo, it is referred to as a "Collection". Thus,a collection is a sequence of granules sharing the same product specification. It typically corresponds to the series of products derived from data acquired by a sensor on board of a satellite and having the same mode of operation. 
+
+In openEO, a back-end offers a set of collections to be processed.  A user can load (a subset of) a collection using a special process, which returns a (spatial) datacube. 
```

Comment:
```suggestion
In openEO, a back-end offers a set of collections to be processed.  A user can load (a subset of) a collection using the `load_collection` process, which returns a raster data cube. 
```

I would not call `load_collection` a special process, it's not very special, it's one of the most common ones :smile:

Reply:
updated. Nevertheless, this was the most special process I found since it never threw me any error 😆